media/js/firefox/sticky-promo.js

```diff
@@ -43,20 +69,39 @@
         };
 
         StickyPromo.show = function (){
-        // Open promo
-            Mzp.StickyPromo.open();
+
+            function openOnScroll(){
+                // Open promo
+                Mzp.StickyPromo.open();
+                document.removeEventListener('scroll', openOnScroll, false);
+            }
+
+            document.addEventListener('scroll', openOnScroll, false);
+
+            var checkForFooterOnScroll = debounce(function() {
+                // If the footer is in the viewport, fade out the promo.
+                // Animate it back in when the footer leaves the viewport
+                // if the user did not dismiss it.
+                if (isInViewport(footer)) {
+                    promo.classList.replace('mzp-a-slide-in', 'mzp-a-fade-out');
```

Comment:
It looks like Internet explorer doesn't support `replace()` on `classList`, so you might have to fallback to using `remove()` and `add()` as separate calls.